infra/clusters/argocd-apps/templates/cluster-compositions/aws-composition.yaml

```diff
@@ -1,14 +1,21 @@
+{{- with .Values.clusters }}
+{{- range . }}
 apiVersion: argoproj.io/v1alpha1
 kind: Application
 metadata:
-  name: crossplane-aws-dev-k8s-packages
-  namespace: argocd
+  name: {{ .name }}
+  namespace: {{ $.Release.Namespace }}
 spec:
-  project: default
+  project: {{ $.Values.project }}
   source:
```

Comment:
Implement multi source repo. by using Sheri's app as example. We need to take app from helm-supporting tools and values from this repo